### PR TITLE
Scaffold archive editorial

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

File: archive-editorial.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    </div>
+
+    <div class="col col6">
+
+<?php
+  while( have_posts() ) {
+    the_post();
+?>
+
+    <?php the_post_thumbnail(); ?>
```
Comment (varffs) [inline archive-editorial.php]: this is still missing the ID to relate the <li> and the image

**Comment by mikeyrayvon** [comment]
ok @varffs now were good

File: js/main.js [hunk 2]
```diff
@@ -10,6 +10,12 @@ Site = {
       _this.onResize();
     });
 
+    if ($('body').hasClass('post-type-archive-editorial')) {
+      $('.archive-title').on('mouseover',function() {
```
Comment (varffs) [inline js/main.js]: this should be a call to `_this.Editorial.init()` or something like that

File: js/main.js [hunk 3]
```diff
@@ -25,11 +31,16 @@ Site = {
       $(this).html(string);
     });
   },
+
+  showArchiveImage: function(id) {
```
Comment (varffs) [inline js/main.js]: this should be in it's own object as a child of `Site`. something like `Site.Editorial`. That way if we scale we don't need to refactor, and code is kept it its own place

File: lib/thumbnail-sizes.php [hunk 4]
```diff
@@ -8,4 +8,5 @@
   add_image_size( 'admin-thumb', 150, 150, false );
   add_image_size( 'opengraph', 1200, 630, true );
   add_image_size( 'logo', 9999, 150, false );
+  add_image_size( 'slide', 1200, 9999, false );
```
Comment (varffs) [inline lib/thumbnail-sizes.php]: should this not have max both dimensions? where does the 1200 come from?

**Comment by varffs** [comment]
Just noticed as well that you have been using a column class that doesn't exist. But its ok we can just change the CSS to match it. `col4` vs `col-4`

**Comment by varffs** [comment]
Oh actually not. Some of the code has the - and some of the code doesnt. So not sure really. I think I prefer without the - so lets do that and I'll update the boiler too

File: archive-editorial.php [hunk 5]
```diff
@@ -35,7 +35,7 @@
     the_post();
 ?>
 
-    <?php the_post_thumbnail('slide', array('data-id' => $post->ID, 'class' => 'archive-image')); ?>
+    <?php the_post_thumbnail( 'full', array('data-id' => $post->ID, 'class' => 'archive-image') ); ?>
```
Comment (varffs) [inline archive-editorial.php]: this means the image can be any amount of pixels big? 4k 5k etc? do the slides fit inside a columized space? if so we can have a image size that max width is the max of its container

Comment (varffs) [inline archive-editorial.php]: ah its inside a col6. so I would set an image size called 'col6' with the width whatever col6 is and no cropping

Comment (mikeyrayvon) [inline archive-editorial.php]: youre confusing me with your comments @varffs cause before you said 

> should this not have max both dimensions? where does the 1200 come from?

but yea col6 width sounds good.  which is what?  i think we should make the site for a slightly larger screen in mind. like 1600px container

Comment (varffs) [inline archive-editorial.php]: well yeah if we gonna do a larger responsive breakpoint then the value set dont matter much. If you want to set a height then I'd choose a ratio and add a image size `col6-2to3` or something like that

**Pull request merged**